### DISCUSSION TITLE
Working again

### DISCUSSION THREAD

**Discussion by anafaggiano** [robaone/yEd-github-chrome-extension#21] [General]
Not sure what happened on my side of things, but the extension is working again, from France and all. Niice

Just a note: it always open the raw git in a new tab before opening the yed editor, where the file is that same raw git address. Not sure it happened before, so I just thought I'd mention it !

**Comment by robaone** [comment]
@anafaggiano this is the normal flow.  Github creates the url for the raw file with a token that is unique so that the data can be retrieved.  The plugin uses that url and sends it to the yEd Online editor.
